bundles/org.openhab.ui.basic/src/main/resources/OH-INF/i18n/basic_fr.properties

```diff
@@ -1,10 +1,41 @@
+ui.config.basic.capitalizeValues.label = Mettre en majuscule les valeurs
+ui.config.basic.capitalizeValues.description = Affiche tous les états de widget en majuscule.
+ui.config.basic.capitalizeValues.option.true = Activer
+ui.config.basic.capitalizeValues.option.false = Désactiver
+ui.config.basic.condensedLayout.label = Mise en page condensée
+ui.config.basic.condensedLayout.description = Lorsque cette option est activée, modifie la mise en page pour que plus de widgets puissent s'afficher à l'écran.
+ui.config.basic.condensedLayout.option.true = Activer
+ui.config.basic.condensedLayout.option.false = Désactiver
+ui.config.basic.defaultSitemap.label = Sitemap par défaut
+ui.config.basic.defaultSitemap.description = La sitemap à afficher si aucun paramètre n'est passé.
+ui.config.basic.enableIconify.label = Activer les icônes Iconify
+ui.config.basic.enableIconify.description = Si cette option est activée, l'interface utilisateur affichera les icônes iconify directement en les téléchargeant à partir d'Internet et en les mettant en cache dans le navigateur Web. Si désactivée, l'interface utilisateur utilisera la méthode standard en requêtant le serveur proxy d'icône et en supposant que les icônes iconify sont installées sur le serveur. La valeur par défaut est désactivée.
+ui.config.basic.enableIconify.option.true = Activer
+ui.config.basic.enableIconify.option.false = Désactiver
+ui.config.basic.enableIcons.label = Activer les icônes
+ui.config.basic.enableIcons.description = Définit si l'interface utilisateur affiche les icônes pour les widgets ou non.
+ui.config.basic.enableIcons.option.true = Activer
+ui.config.basic.enableIcons.option.false = Désactiver
+ui.config.basic.theme.label = Thème
+ui.config.basic.theme.description = Définit le thème de l'interface utilisateur.
+ui.config.basic.theme.option.default = Par défaut
+ui.config.basic.theme.option.dark = Sombre
+ui.config.basic.webAudio.label = Audio Web
+ui.config.basic.webAudio.description = Jouer de l'audio avec le lecteur audio Web.
+ui.config.basic.webAudio.option.true = Activer
+ui.config.basic.webAudio.option.false = Désactiver
+
+# service
+
+service.ui.basic.label = Basic UI
+
 # messages
 
-main.offline-msg = Non connecté: en attente d'une connexion disponible
-main.long-polling-mode-msg = Échec de l'abonnement SSE ({error}): exécution en mode secours
+main.offline-msg = Hors ligne \: en attente de la connexion
```

Comment:
I proposed in Crowdin a new translation, the original one but with the right encoding.